recipes/qt/6.x.x/conanfile.py

```diff
@@ -1137,6 +1137,7 @@ def _create_plugin(pluginname, libname, type, requires):
                 self.cpp_info.components["qtCore"].system_libs.append("ws2_32")  # qtcore requires "WSAStartup " which is in "Ws2_32.Lib" library
                 self.cpp_info.components["qtNetwork"].system_libs.append("dnsapi")  # qtnetwork from qtbase requires "DnsFree" which is in "Dnsapi.lib" library
                 self.cpp_info.components["qtNetwork"].system_libs.append("iphlpapi")
+                self.cpp_info.components["qtNetwork"].system_libs.extend(["winhttp", "secur32"])
```

Comment:
interesting, why it was needed before...

Reply:
they were loaded at run-time: https://github.com/qt/qtbase/commit/2526df506bda99f9497e83a4e6820557a722660e#diff-bfae164fc302cc539568d38eca190a906cc42f245a02afc8c5e9821951ab205bL490  https://github.com/qt/qtbase/commit/2526df506bda99f9497e83a4e6820557a722660e#diff-01c557d3e7d8a8dc706be8b50972f604837065084788aa7f1dc5b868db372967L1563
These libs exist since windows XP, and qt6 does not support windows before 10, so it's fine to add them to system_libs.